generators/server/templates/pom.xml.ejs

```diff
@@ -224,6 +227,11 @@ _%>
             <artifactId>logback-classic</artifactId>
             <version>${logback.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+            <version>${logback.version}</version>
```

Comment:
```suggestion
            <version>${javax-inject.version}</version>
```

Reply:
Gets things further along to where we can at least see some specific failing tests. :)

Out of curiosity, know what is requiring the javax.inject dependency? It's not any of the generated code and it's not anything from Micronaut.

Reply:
It was a failure in CI: https://github.com/jhipster/generator-jhipster-micronaut/actions/runs/5859326680/job/15885088988